cmd/evm/internal/t8ntool/execution.go

```diff
@@ -47,16 +47,17 @@ type Prestate struct {
 // ExecutionResult contains the execution status after running a state test, any
 // error that might have occurred and a dump of the final state if requested.
 type ExecutionResult struct {
-	StateRoot   common.Hash           `json:"stateRoot"`
-	TxRoot      common.Hash           `json:"txRoot"`
-	ReceiptRoot common.Hash           `json:"receiptsRoot"`
-	LogsHash    common.Hash           `json:"logsHash"`
-	Bloom       types.Bloom           `json:"logsBloom"        gencodec:"required"`
-	Receipts    types.Receipts        `json:"receipts"`
-	Rejected    []*rejectedTx         `json:"rejected,omitempty"`
-	Difficulty  *math.HexOrDecimal256 `json:"currentDifficulty" gencodec:"required"`
-	GasUsed     math.HexOrDecimal64   `json:"gasUsed"`
-	BaseFee     *math.HexOrDecimal256 `json:"currentBaseFee,omitempty"`
+	StateRoot      common.Hash           `json:"stateRoot"`
+	TxRoot         common.Hash           `json:"txRoot"`
+	ReceiptRoot    common.Hash           `json:"receiptsRoot"`
+	LogsHash       common.Hash           `json:"logsHash"`
+	Bloom          types.Bloom           `json:"logsBloom"        gencodec:"required"`
+	Receipts       types.Receipts        `json:"receipts"`
+	Rejected       []*rejectedTx         `json:"rejected,omitempty"`
+	Difficulty     *math.HexOrDecimal256 `json:"currentDifficulty" gencodec:"required"`
+	GasUsed        math.HexOrDecimal64   `json:"gasUsed"`
+	BaseFee        *math.HexOrDecimal256 `json:"currentBaseFee,omitempty"`
+	WithdrawalRoot *common.Hash          `json:"withdrawalRoot,omitempty"`
```

Comment:
Just a small nitpick here is that the `header` struct in `cmd/evm/internal/t8ntool/block.go` takes `withdrawalsRoot` as json field (compared to `withdrawalRoot` here), so it might cause confusion/extra refactoring on the testing tooling.
It might be worth it to have `withdrawalsRoot` everywhere.

Reply:
+1 on `withdrawalsRoot` as this is what the EIP uses, and also is grammatically aligned -- it is the root of possibly many withdrawal**s**, not just a single one

Reply:
Thanks for catching this, I'll update.

Reply:
Fixed.